support/runner-script.py

```
@@ -98,6 +107,34 @@ def _download_single(name: str, hash: str) -> bytes:
 		return f.read()
 
 
+def _nix_preload(cur_dst: Path, name: str, hash: str):
+	if not args.nix_preload:
+		return
+	import subprocess
+
+	print(f'info: preloading {cur_dst} into nix store as genvm_runner_{name}_{hash}')
+
+	subprocess.run(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)